<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphcalc</class>
 <widget class="QDialog" name="graphcalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>571</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>669</width>
    <height>571</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Калькулятор SmartCalc v1.0 - Построение графика</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #F7F8FB;</string>
  </property>
  <widget class="QLineEdit" name="input_max_x">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QLineEdit{
     background: #ffffff;
     border: 2px solid #F7F8FB;
     color: #424242;
     border-radius: 16px;
     font-size: 18px;
     font-weight: 600;
     padding-left: 15px;
     padding-right: 15px;
     }


     QLineEdit::hover{
     background: #FCFCFE;
     }

     QLineEdit:pressed{
     background: #EFF7FF;
     }


    </string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max: X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min: X</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_min_x">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QLineEdit{
     background: #ffffff;
     border: 2px solid #F7F8FB;
     color: #424242;
     border-radius: 16px;
     font-size: 18px;
     font-weight: 600;
     padding-left: 15px;
     padding-right: 15px;
     }


     QLineEdit::hover{
     background: #FCFCFE;
     }

     QLineEdit:pressed{
     background: #EFF7FF;
     }


    </string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min: Y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_min_y">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QLineEdit{
     background: #ffffff;
     border: 2px solid #F7F8FB;
     color: #424242;
     border-radius: 16px;
     font-size: 18px;
     font-weight: 600;
     padding-left: 15px;
     padding-right: 15px;
     }


     QLineEdit::hover{
     background: #FCFCFE;
     }

     QLineEdit:pressed{
     background: #EFF7FF;
     }


    </string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max: Y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_max_y">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QLineEdit{
     background: #ffffff;
     border: 2px solid #F7F8FB;
     color: #424242;
     border-radius: 16px;
     font-size: 18px;
     font-weight: 600;
     padding-left: 15px;
     padding-right: 15px;
     }


     QLineEdit::hover{
     background: #FCFCFE;
     }

     QLineEdit:pressed{
     background: #EFF7FF;
     }


    </string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_draw_graph">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
     background: #ffffff;
     color: #7CC9FF;
     border-radius: 16px;
     font-size: 18px;
     font-weight: 600;
     }

     QPushButton::hover{
     background: #FCFCFC;
     }

     QPushButton:pressed{
     background: #EFF7FF;
     }</string>
   </property>
   <property name="text">
    <string>draw</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget_graph" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>671</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выражение:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="expressionValue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QLineEdit{
     background: #ffffff;
     border: 2px solid #F7F8FB;
     color: #424242;
     border-radius: 16px;
     font-size: 18px;
     font-weight: 600;
     padding-left: 15px;
     padding-right: 15px;
     }


     QLineEdit::hover{
     background: #FCFCFE;
     }

     QLineEdit:pressed{
     background: #EFF7FF;
     }


    </string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="xValue">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QLineEdit{
     background: #ffffff;
     border: 2px solid #F7F8FB;
     color: #424242;
     border-radius: 16px;
     font-size: 18px;
     font-weight: 600;
     padding-left: 15px;
     padding-right: 15px;
     }


     QLineEdit::hover{
     background: #FCFCFE;
     }

     QLineEdit:pressed{
     background: #EFF7FF;
     }


    </string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>lib/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
